.wp-env.json

```diff
@@ -0,0 +1,9 @@
+{
+	"core": null,
+	"plugins": [ "." ],
+	"env": {
+		"tests": {
+			"phpVersion": "7.4"
+		}
+	}
+}
```

Comment:
I think the indentation here is incorrect, it needs to follow https://github.com/10up/plugin-check/blob/trunk/.editorconfig#L23.